src/wallet/wallet2.cpp

```diff
@@ -13046,6 +13055,77 @@ void wallet2::throw_on_rpc_response_error(const boost::optional<std::string> &st
   THROW_WALLET_EXCEPTION_IF(*status != CORE_RPC_STATUS_OK, tools::error::wallet_generic_rpc_error, method, m_trusted_daemon ? *status : "daemon error");
 }
 //----------------------------------------------------------------------------------------------------
+
+bool wallet2::save_to_file(const std::string& path_to_file, const std::string& raw, bool is_hex) const
+{
+  if (is_hex || m_export_format == ExportFormat::Raw)
+  {
+    return epee::file_io_utils::save_string_to_file(path_to_file, raw);
+  }
+
+  std::string h = HEX_OUTPUT_MAGIC + "\n";
+  h.append(epee::string_tools::buff_to_hex_nodelimer(raw));
```

Comment:
`base58` or `base64` will have smaller output. `base58` is provided internally. `EVP_EncodeBlock` and `EVP_DecodeBlock` from OpenSSL provide to/from `base64` with newlines every 64 characters. The OpenSSL decode function is also useful because it strips all whitespace.

Reply:
Sadly, the wallet does not use OpenSSL currently, and adding such dependency for something so trivial doesn't make sense.

Using `base58` is a good idea though, IMO better than the hexdump we've done previously. Let me try to implement it. Thanks!

Reply:
The wallet links against `libcrypto` and `libssl` transitively from `epee` SSL code.

Reply:
The dependency to OpenSSL could be dropped  if the wallet switches to another SSL library, drops SSL altogether, or builds SSL as an optional build option? The third scenario seems the most likely, but the code would require some serious work to get to that.